src/plots/gl2d/index.js

```diff
@@ -27,6 +30,21 @@ exports.attrRegex = constants.attrRegex;
 
 exports.attributes = require('../cartesian/attributes');
 
+// gl2d uses svg axis attributes verbatim, but overrides editType
+// this could potentially be just `layoutAttributes` but it would
+// still need special handling somewhere to give it precedence over
+// the svg version when both are in use on one plot
```

Comment:
> but it would still need special handling somewhere to give it precedence over the svg version when both are in use on one plot

Can you elaborate on this? Depending on how complicated that needed _special handling_, I might vote for trying to :hocho: those *AttrOverride` fields - which feels like a layer of abstraction we could avoid.

Reply:
That was basically in response to [this block](https://github.com/plotly/plotly.js/blob/11bb997e78128b9f47db2ed392982f82a026e589/src/plot_api/plot_api.js#L2223-L2227) that says "any axis attribute when the plot has gl2d needs to be treated as `doplot`." - But yeah, not the nicest structure, I'm open to suggestions.